src/test/rustdoc-gui/src/test_docs/build.rs

```diff
@@ -0,0 +1,16 @@
+//! generate 2000 constants for testing
+
+use std::fs::write;
+
+fn main() -> std::io::Result<()> {
+    let out_dir = std::env::var("OUT_DIR").expect("OUT_DIR is not defined");
+
+    let mut output = String::new();
+    for i in 0..2000 {
```

Comment:
1200 should be enough I think. Since we rebuild doc every time, maybe better to not write too much.

Reply:
the grid-2 spec talks about -10k to +10k cells, it is currently a blink-engine thing to clamp at 1k. In some of their examples I saw them using 1.5k lines so I guessed 2k.
Switching to a fixed file instead of generating it all the time would reduce the amount of writing

Reply:
Then let's keep 2k and generate a file.